pkgs/applications/video/kodi/addons/horus/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, buildKodiAddon, fetchzip, kodi-six }:
+
+buildKodiAddon rec {
+  pname = "horus";
+  namespace = "script.module.horus";
+  version = "1.1.9";
+  src = fetchzip {
+    url = "https://github.com/gtkingbuild/Repo-GTKing/raw/master/omega/zips/${namespace}/${namespace}-${version}.zip";
```

Comment:
🫠

this doesn't get automatically updated by our scripts then
i can't find a repo dedicated for just this addon... i am guessing it doesn't exist?